### PR TITLE
Redux toolkit implementation

### PR DESCRIPTION
**Pull request by tinatsina**
## Summary
- Applied basic redux implementations
- Created two separate files
    - books
    - categories
- added central configurations for redux

## Changelog
- `feature` ---> `development`

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const initialState = [];
+
+export const ADD_BOOK = (data) => ({ type: 'ADD_BOOK', data });
+export const DEL_BOOK = (data) => ({ type: 'DEL_BOOK', data });
```
Comment (Davidosky007) [inline src/redux/books/books.js:4]: * Kindly use the [ducks pattern ](https://github.com/erikras/ducks-modular-redux) for creating your action types as shown below 
image

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const initialState = [];
+
+export const CHECK_STATUS = (data) => ({ type: CHECK_STATUS, data });
```
Comment (Davidosky007) [inline src/redux/categories/categories.js:3]: * Also use Ducks pattern here for your action types as i mentioned above ⬆️

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @tinatsina 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* Well written PR description 👍 
* Good looking readme file 👍 
* Proper use of Gitflow 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brytebee — APPROVED** [review]
Hi, @tinatsina👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 

##

###  **Optional Suggestions:**
- N/A

------

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**